internal/workload/podspec_updates.go

```diff
@@ -475,20 +478,44 @@ func (s *updateState) updateContainer(p *cloudsqlapi.AuthProxyWorkload, wl Workl
 		params := map[string]string{}
 
 		// if it is a TCP socket
+		if inst.UnixSocketPath == "" {
+
+			port := s.useInstancePort(p, inst)
+			params["port"] = fmt.Sprint(port)
+			if inst.HostEnvName != "" {
+				s.addWorkloadEnvVar(p, inst, corev1.EnvVar{
+					Name:  inst.HostEnvName,
+					Value: "127.0.0.1",
+				})
+			}
+			if inst.PortEnvName != "" {
+				s.addWorkloadEnvVar(p, inst, corev1.EnvVar{
+					Name:  inst.PortEnvName,
+					Value: fmt.Sprint(port),
+				})
+			}
+		} else {
+			// else if it is a unix socket
+			params["unix-socket-path"] = inst.UnixSocketPath
```

Comment:
Where are we using `params`?

Reply:
Do you think we should make these parameters configurable in the environment in the proxy?

Reply:
Yes, instance configuration should also be converted to EnvVars. I want to do a clean "feat: add unix sockets" PR without changing much other stuff first.

Reply:
Right now the Proxy doesn't support configuring instance-level query params with env-vars. So we think that's valuable, we'll have to start with the proxy first.